src_c/mask.c

```diff
@@ -708,6 +708,7 @@ set_from_threshold(SDL_Surface *surf, bitmask_t *bitmask, int threshold)
     Uint8 bpp = format->BytesPerPixel;
     Uint8 *pixel = NULL;
     Uint8 rgba[4];
+    rgba = 0;
```

Comment:
array init

Reply:
fixed by checking for 'array(' by ' '.joining orig_type (which was a list, not a string)